fedcred/common.py

```diff
@@ -49,13 +49,19 @@ def read_config():
 
 
 def get_user_credentials(prompt=None):
-    if prompt is None:
+    if "AWS_USERNAME" in os.environ:
```

Comment:
Username is generalized here because this is the username that is referenced at your identity provider. Calling it `AWS_USERNAME` confuses its usage.